### ISSUE TITLE
[bug]: incorrect validation multiple rules after v0.6.7

### ISSUE THREAD

**Issue by Scrum** [icebob/fastest-validator#27]
```js
let schema = {
  value: [
    { type: "string", min: 3, max: 255 },
    { type: "boolean" }
  ]
};

let obj = { value: undefined };

let res = check(obj);

// return => true
```

**Comment by icebob** [comment]
@mauricedoepke do you have idea?

**Comment by mauricedoepke** [comment]
@icebob I will look into it on the weekend

**Comment by icebob** [comment]
Thanks

**Comment by Scrum** [comment]
@mauricedoepke Hi, is there progress?

**Comment by icebob** [comment]
Fixed, released in 0.6.10.
